### DISCUSSION TITLE
Problem loading from url  errors encountered

### DISCUSSION THREAD

**Discussion by RickLovie** [berrystephenw/OctoText#69] [General]
Hi I loaded octotext when it first came out on one of my machines ( it's great I love it)  I went back to load it on another machine through the repertoire but it was not there anymore, so I went to load it from the url and when it installs I get an error 
ERROR: Package 'OctoPrint-OctoText' requires a different Python: 2.7.16 not in '>=3, <4'
Error!
Could not parse output from pip, see plugin_pluginmanager_console.log for generated output 

 Then a green box pops up and says X INSTALL UNKNOWN   
 Any ideas  I love using it on my other machine
  Thanks in advance

**Comment by berrystephenw** [comment]
Hi Rick, The problem is your second Pi is running an older version of Python. OctoText requires Python 3 and that is what the installation failure is trying to tell you. Please check out this discussion on how to upgrade the distribution so you can run newer plugins that aren't backwards compatible: https://github.com/berrystephenw/OctoText/discussions/1#discussioncomment-557346

I wish I could make it compatible with the older Python 2 code, but I have limited time and it would involve a lot more testing! I'm glad it works for you on your other machine ;)

**Reply by RickLovie** [reply]
Thank you, my Pi had both versions of Python on it 2 and 3 and I was able to update octoprint to python3 so all is good!
